tests/test_maintel_close_loop_lsstcam.py

```diff
@@ -0,0 +1,148 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+import asyncio
+import random
+import unittest
+
+import numpy as np
+from lsst.ts import standardscripts
+from lsst.ts.observatory.control.utils.enums import ClosedLoopMode
+from lsst.ts.standardscripts.maintel import CloseLoopLSSTCam
+
+random.seed(47)  # for set_random_lsst_dds_partition_prefix
+
+
+class TestCloseLoopLSSTCam(
+    standardscripts.BaseScriptTestCase, unittest.IsolatedAsyncioTestCase
+):
+    async def basic_make_script(self, index):
+        self.script = CloseLoopLSSTCam(index=index, add_remotes=False)
```

Comment:
Instead of using `add_remotes`, set the  value of the camera and TCS classes here with a `DryTest` usage, e.g.;

```
self.script = CloseLoopLSSTCam(index=index)
self.script.mtcs = MTCS(
                domain=self.script.domain,
                intended_usage=MTCSUsages.DryTest,
                log=self.script.log,
            )
self.script._lsstcam = LSSTCam(
                domain=self.script.domain,
                intended_usage=LSSTCam.DryTest,
                log=self.script.log,
            )
```